newIDE/app/src/ResourcesList/index.js

```diff
@@ -79,6 +83,38 @@ export default class ResourcesList extends React.Component<Props, State> {
     this.props.onDeleteResource(resource);
   };
 
+  _scanForNewResources = () => {
+    const resourcesList = this
+    const project = this.props.project
+    const resourcesManager = project.getResourcesManager()
+    console.log('Scanning the project folder for new resources...');
+    const projectPath = path.dirname(project.getProjectFile());
+
+    const getDirectories = (src, callback) => {
+      glob(src + '/**/*.png', callback);
+    };
+    getDirectories(projectPath, function (err, res) {
+      if (err) {
+        console.log('Error loading ', err);
+      } else {
+        res.forEach((pathFound)=>{
+          // should use relative path for resource name here!
+          let fileName = String(pathFound).split('/').pop();
+          /// todo: fix gdevelop's naming of imported resources not including subfolder case
```

Comment:
So, it's up to you to decide the convention of resource naming. GDevelop will only make sure that two resources can't have the same name (otherwise there is no way to know which one you want when using it on an object).
Note that resource names can be anything, including a path with slashes. So you could name the resource with the relative path to the project (`C:\Folder\MyProject\player\idle.png` would become `player/idle.png`).

Sounds like the most obvious/intuitive way, what do you think?

Reply:
Please note that a backslash "\" in the path would cause trouble opening a project on Linux/OSX. But Windows also understands forward slashes "/" in paths so it should be enough to replace "\" with "/" in the identifier string.

Reply:
good point, I will address this by using path.basename instead

Reply:
Sorry missed your comment, relative path should be used yeah - I think that will give the best result and it's simple to use and understand.

Reply:
@4ian I also think of renaming the 'Delete' menu option to 'Remove'. It doesn't actually delete the resource file - it just removes it from the project. What do you think?

Reply:
This is a good idea!